Android/app/src/main/java/live/dittolive/chat/SplashActivity.kt

```diff
@@ -48,15 +48,39 @@ class SplashActivity : AppCompatActivity() {
   }
 
   private fun setupDitto() {
-    val androidDependencies = DefaultAndroidDittoDependencies(applicationContext)
     DittoLogger.minimumLogLevel = DittoLogLevel.DEBUG
+
+    val androidDependencies = DefaultAndroidDittoDependencies(applicationContext)
     ditto = Ditto(
-      androidDependencies,
-      DittoIdentity.OfflinePlayground(androidDependencies, BuildConfig.DITTO_APP_ID)
+      dependencies = androidDependencies,
+      identity = resolveIdentity(androidDependencies)
     )
-    ditto.setOfflineOnlyLicenseToken(BuildConfig.DITTO_LICENSE_TOKEN)
+    ditto.setOfflineOnlyLicenseToken()
     // Disable sync with V3
-        ditto.disableSyncWithV3()
+    ditto.disableSyncWithV3()
     ditto.startSync()
   }
+
+  /**
+   * Returns [DittoIdentity.OnlinePlayground] for Debug builds,
+   * [DittoIdentity.OfflinePlayground] otherwise.
+   */
+  private fun resolveIdentity(androidDependencies: DefaultAndroidDittoDependencies): DittoIdentity {
+    if (BuildConfig.DEBUG) return DittoIdentity.OnlinePlayground(
+      dependencies = androidDependencies,
+      appId = BuildConfig.DITTO_APP_ID,
+      token = BuildConfig.DITTO_LICENSE_TOKEN
```

Comment:
The Playground token will need to be used here. `BuildConfig.DITTO_LICENSE_TOKEN` refers to the offline-only license token which is different from the Playground token.

Reply:
@texasRanger09 I agree that `DITTO_LICENSE_TOKEN` is kind of confusing for the Online Playground.
Current configuration in gradle looks like:
```
debug {
            def credsFile = rootProject.file("secure\\debug_creds.properties")
            ...
            buildConfigField('String', 'DITTO_LICENSE_TOKEN', prop['DITTO_LICENSE_TOKEN'])
}
```

What do you think about making a change to something like:
```
debug {
            def credsFile = rootProject.file("secure\\debug_creds.properties")
            ...
            buildConfigField('String', 'DITTO_PLAYGROUND_TOKEN', prop['DITTO_PLAYGROUND_TOKEN'])
}
```

And of-course with the corresponding changes in the README.md

Reply:
Actually I made a typo in the previous message.
We may want to have a generic `DITTO_TOKEN` for "debug" and "release" but the following properties on the credentials files.

`debug_creds.properties` 
DITTO_PLAYGROUND_TOKEN=...

`release_creds.properties` 
DITTO_LICENSE_TOKEN=...
 

**build.gradle**
```
debug {
            def credsFile = rootProject.file("secure\\debug_creds.properties")
            ...
            buildConfigField('String', 'DITTO_TOKEN', prop['DITTO_PLAYGROUND_TOKEN'])
}

debug {
            def credsFile = rootProject.file("secure\\release_creds.properties")
            ...
            buildConfigField('String', 'DITTO_TOKEN', prop['DITTO_LICENSE_TOKEN'])
}
```

Reply:
That looks right. I opened an issue #82 to address the confusing environment variable names. The Android variable names should be updated to match the iOS naming convention.

Reply:
Cool, I will implement the suggested changes.
Thank for the review.

Reply:
> We may want to have a generic `DITTO_TOKEN` for "debug" and "release"

The `App Id` is needed for both online and offline playground. However, the online playground token is not needed when using the offline playground identity and the offline license token is not needed when using the online playground identity.

Reply:
You are correct, `BuildConfig.DITTO_TOKEN` will be the name generated by the Android plugin at compile time, the value of this variable will be set based on the build type:

**debug** will use the env var `DITTO_PLAYGROUND_TOKEN` stored in file `secure\\debug_creds.properties`
**release** will use the env var `DITTO_LICENSE_TOKEN ` stored in file `secure\\release_creds.properties`

I think that once #82 is ready for implementation, then Android can make use of only one property file.